tests/queries/0_stateless/02496_format_datetime_in_joda_syntax.sql

```diff
@@ -0,0 +1,142 @@
+-- { echoOn }
```

Comment:
Bonus points for also applying the same test cases on `Date` and `DateTime64` inputs (right now, `DateTime` and `Date32` are checked).